app/Controllers/Tenant/CategoriesController.php

```
@@ -0,0 +1,42 @@
+
+namespace App\Controllers\Tenant;
+
+use App\Controllers\BaseController;
+use App\Enums\PostState;
+use App\Models\CategoryModel;
+use App\Models\PostModel;
+use CodeIgniter\Exceptions\PageNotFoundException;
+
+class CategoriesController extends BaseController
+{
+    public function posts(string $tenantSlug, string $categorySlug): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$tenantSlug'. (undefined)

(UnusedFormalParameter)